appveyor.yml

```diff
@@ -2,21 +2,28 @@ version: 1.0.{build}
 branches:
   only:
   - windows_port
+clone_depth: 5
```

Comment:
@dawagner said:
> Unfortunately, this can break `git describe --tags` - which is used in the main CMakeLists.txt. This is currently only used for documentation generation but I'm afraid it will lead to "interesting bugs" when we want to use it for more.
Well travis has a 50 history so I should maybe use the same. What is your opinion ?